interview-questions.md

```diff
@@ -9,60 +9,59 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is `this` in JavaScript?
 
-  Your answer: 'This' in JavaScript is a means of identifying and retaining a particular value. 
-  This can be associated to a state or a value.
+  Your answer:
 
-  Researched answer: In JavaScript, when calling a function 'this' is the way to set a value. 'This' shows the ownership of a value to its respective function.
+  Researched answer:
 
 
 
 2. What is React? Why would you use it?
 
-  Your answer: React is a Javascript library that improves workflow by importing various components to speed up the building process of coding. You woulds use React to build user interfaces.
+  Your answer:
 
-  Researched answer: React is a component-based JavaScript library that used to build interactive components on a user interface. React is specific is building single page applications. One  would use React to view the changes being made without the need to reload the page.
+  Researched answer:
 
 
 
 3. Which lifecycle method is required in a React class component?
 
-  Your answer: The lifecycle method that is required in a React class component is the render method. The other methods are mounts, unmounts and updates.
+  Your answer:
 
-  Researched answer: Render is required because it is the only way to refresh the changes made to the User Interface.
+  Researched answer:
 
 
 
 4. What is JSX? What is one notable difference between HTML and JSX?
 
-  Your answer: Jsx is a JavaScript XML and is a component that is used to write code using HTML in React.A notable difference is that JSX uses fragments in React and in HTML a script tag is required.
+  Your answer:
+
+  Researched answer:
 
-  Researched answer: JSX is a component of JavaScript that is used with React to build
-  by enabling the integration of multiple languages respective to their tag.
 
 
 5. What is yarn? What file(s) are modified in a React application when you run yarn?
 
-  Your answer: Yarn is a package created by Meta that is used with React to run and test code. When Yarn is ran, the necessary dependencies are checked, and if necessary updated.
+  Your answer:
 
-  Researched answer: Yet Another Resource Manager(Yarn) is an open-source project manager and package manager that was developed by Facebook. The yarnrc file gets modified and allows configurational changes and adjustments to the Yarn features.
```

Comment:
Yes, yarn will connect dependencies. The yarn.lock, package.json, and node_nodules will be updated.